spark-commons/src/test/scala/za/co/absa/spark/commons/errorhandling/implementations/ErrorHandlingFilterRowsWithErrorsTest.scala

```diff
@@ -87,4 +89,35 @@ class ErrorHandlingFilterRowsWithErrorsTest extends AnyFunSuite with SparkTestBa
 
     assert(result == expected)
   }
+
+  test("errorColumnType should return a BooleanType") {
+    val errorColumn: ErrorColumn = ErrorHandlingFilterRowsWithErrors.putErrorToColumn(
+      "Test error 1", 1, "This is a test error", Some(errColName))
+
+    val testDf = emptyDf.withColumn(errColName, errorColumn.column)
+    val expectedResults = testDf.col(errColName).expr.dataType
+    val expval = testDf.schema.fields
+
+    val results = ErrorHandlingFilterRowsWithErrors.errorColumnType
```

Comment:
Just a proposal: `results` -> `actualType`

Reply:
`results` is the actual results that the tested method return. What if we call it `actualResults` or actualType`